File: 백준/Bronze/10952. A＋B － 5/A＋B － 5.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

let res = [];
input.map((testCase) => {
  const [A, B] = testCase.split(" ");
  testCase !== "0 0" && res.push(parseInt(A) + parseInt(B));
});

console.log(res.join("\n"));

File: 백준/Bronze/10952. A＋B － 5/A＋B － 5.swift
var res: [String] = []
while let line = readLine() {
  if (line == "0 0") {
    break
  }

  let input = line.split(separator: " ").map{ Int($0)! }
  res.append("\(input[0] + input[1])")
}
print(res.joined(separator: "\n"))